librz/include/rz_util/rz_str.h

```diff
@@ -9,6 +9,9 @@
 extern "C" {
 #endif
 
+static const char *rz_str_empty = "";
```

Comment:
is it alright to have variables in a .h file? Why not put it inside the `rz_str_get*` functions? That way, I think, they would be allocated only by the modules that actually use those functions instead of anyone that includes this header.